docs/ECLLanguageReference/ECLR_mods/WkFlo-STORED.xml

```diff
@@ -106,14 +108,22 @@
 
             <entry>An integer expression defining the sequential location of
             the input box. These can be sparse values (e.g., 100, 200, 300) to
-            allow insertion of new inputs in the future. </entry>
+            allow insertion of new inputs in the future.</entry>
           </row>
 
           <row>
             <entry><emphasis role="bold">NOINPUT</emphasis></entry>
 
             <entry>Optional. If NOINPUT is specified, the field is not
-            displayed on the Web form in WsECL. </entry>
+            displayed on the Web form in WsECL.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">PASSWORD</emphasis></entry>
+
+            <entry>Optional. If PASSWORD is specified, a password entry box is
+            used on the Web form in WsECL and the field's supplied value is
+            not displayed while entering it.</entry>
```

Comment:
The value is also hidden when viewing the stored values in the workunit through EclWatch (or command line).  Also when extracting the xml, etc.